executor/show_test.go

```diff
@@ -126,52 +126,34 @@ func (s *testSuite) TestShowVisibility(c *C) {
 	tk.MustExec(`create user 'show'@'%'`)
 	tk.MustExec(`flush privileges`)
 
+	tk1 := testkit.NewTestKit(c, s.store)
 	se, err := tidb.CreateSession(s.store)
 	c.Assert(err, IsNil)
 	c.Assert(se.Auth(`show@%`, nil, nil), IsTrue)
+	tk1.Se = se
```

Comment:
NewTestKit create a new session, which doesn't call `Auth` , so that session won't check privilege. 
I need a session that check privilege for testing, so I create a new one and assign it to `tk1`.